docker-compose.yml

```diff
@@ -50,7 +50,7 @@ services:
       - "9984"
     command: bigchaindb start
   
-  bdb-mdb:
+  bdb:
```

Comment:
That's my bad. I was looking at a different version of the docs, not from this PR!
Approved!